man/mkosi.1

```diff
@@ -769,6 +769,16 @@ Takes the same values as \f[C]--source-file-transfer\f[R] except
 \f[C]mount\f[R].
 By default, sources are not copied into the final image.
 .TP
+\f[B]\f[CB]--source-file-resolve-symlinks\f[B]\f[R]
```

Comment:
I think we really need to make a note that, this file is autogenerated from `mkosi.md`, so that people don't go through the pain of editing raw roff :)

Reply:
Damn! pandoc adds a note at the top which I totally missed. Though I would wonder if an automatically generated file should be tracked in Git... anyhow, I have added the changes to mkosi.md